test/e2e_node/summary_test.go

```diff
@@ -90,8 +90,8 @@ var _ = framework.KubeDescribe("Summary API [NodeConformance]", func() {
 					"StartTime": recent(maxStartAge),
 					"CPU": ptrMatchAllFields(gstruct.Fields{
 						"Time":                 recent(maxStatsAge),
-						"UsageNanoCores":       bounded(10000, 2E9),
-						"UsageCoreNanoSeconds": bounded(10000000, 1E15),
+						"UsageNanoCores":       bounded(10000, 2e9),
```

Comment:
None of the changes here seem related. Could you revert? Just to simplify reviews.

Reply:
Updated as per recommendation